core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -508,6 +508,11 @@
   TSERV_SUMMARY_RETRIEVAL_THREADS("tserver.summary.retrieval.threads", "10", PropertyType.COUNT,
       "The number of threads on each tablet server available to retrieve"
           + " summary data, that is not currently in cache, from RFiles."),
+  TSERV_LASTLOCATION_UPDATE_TIME("tserver.lastlocation.update.time", "1000000", PropertyType.COUNT,
```

Comment:
Without looking at the code (and not being familiar with the particulars) - I find this description / value confusing.  The property is named TIME, but you are using a PropertyType.COUNT.  At first, I wondered why you didn't specify the time as 15m or 900s, or whatever human readable value.

Reply:
That's a fair point. I will look into giving a more readable description. As for value, that is what I used for testing. I should actually set it to 0 for the default and allow the user to change it if they wish to use the feature.

Reply:
We have a `PropertyType.TIMEDURATION` that would be better, and which would allows human-readable configuration, such as '5s'.

Reply:
I used that for my other property. Reason why this is count is because it's used in a comparison between the difference of two longs. It was just a poor name and value for it.